<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConnectUI</class>
 <widget class="QWidget" name="ConnectUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>165</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>请求连接……</string>
  </property>
  <widget class="QPushButton" name="button_connect">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>110</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_ip">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>opc.tcp://localhost</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_ip">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="button_clear">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>160</width>
     <height>35</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_port">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>65</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="edit_port">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>65</y>
     <width>240</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>4840</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_msg">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>90</y>
     <width>100</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_clear</sender>
   <signal>clicked()</signal>
   <receiver>edit_ip</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>109</y>
    </hint>
    <hint type="destinationlabel">
     <x>254</x>
     <y>54</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_clear</sender>
   <signal>clicked()</signal>
   <receiver>edit_port</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>259</x>
     <y>84</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
